engine/class_modules/apl/apl_demon_hunter.cpp

```diff
@@ -44,9 +44,21 @@ std::string rune( const player_t* p )
 
 std::string temporary_enchant( const player_t* p )
 {
-  return ( ( p->true_level >= 61 ) ? "main_hand:buzzing_rune_3/off_hand:buzzing_rune_3" :
-           ( p->true_level >= 51 ) ? "main_hand:shaded_sharpening_stone/off_hand:shaded_sharpening_stone" :
-           "disabled" );
+  std::string shadowlands_temporary_enchant = "main_hand:shaded_sharpening_stone/off_hand:shaded_sharpening_stone";
```

Comment:
Think this would probably just be easier to maintain with the redundant strings and just have the distinct section for Havoc/Vengeance even if it's a little duplicated.